dags/collisions_replicator_transfer.py

```diff
@@ -12,8 +12,12 @@
 #This script does things with those operators:
 #1) uses ACC to update acc_safe_copy
 #2) updates the mat view called collisions_no, which generates ids needed to...
-#3) update the events and involved mat views 
-#4) throws a sassy slack alert message when it fails
+#3) updates the events_cursed and involved_cursed mat views
```

Comment:
We will need to update this DAG with Gabe's new slack username thingy

Reply:
see this : https://github.com/CityofToronto/bdit_data-sources/pull/597
and this bug fix  https://github.com/CityofToronto/bdit_data-sources/pull/633/files PR for `name`

Reply:
Implemented!